docs/setup_visualstudio.md

```diff
@@ -209,54 +209,53 @@ Visual Studio provides an address sanitizer with bounds checking automatically w
 
 </div>
 
-### Input redirection
+### Arguments and options
 <div class="primer-spec-callout info" markdown="1">
 Skip this subsection your first time through the tutorial.  You can come back to it.
 </div>
 
-If you're unfamiliar with input redirection, first read the CLI tutorial section on [input redirection](cli.html#input-redirection-).
-
-Without input redirection, here's how to type input in the Visual Studio command line.  Notice that when we run, a window pops up that accepts user input.
+Arguments and options are inputs to a program typed at the command line.  Here's an example from EECS 280 Project 5:
```

Comment:
Probably should do a `grep` or `sed` for this

Reply:
Fixed via c970e4d.